tests/express.test.ts

```diff
@@ -0,0 +1,112 @@
+import { setLocalHeaders, setBasicSecurity, setBodyParser, setRateLimiter } from '../src/lib/express';
+const request = require('supertest-as-promised');
+const redisMock = require('redis-mock');
+const express = require('express');
+
+describe('Express', () => {
+  describe('#setLocalHeaders', () => {
+    let app;
+    beforeEach(() => {
+      app = express();
+    });
+
+    test('app should have Access-Control headers', async () => {
+      setLocalHeaders(app, '*');
+      app.use('/', (req, res) => res.status(200).send('Welcome'));
+
+      const { headers } = await request(app).get('/');
+      expect(headers).toHaveProperty('access-control-allow-origin');
+      expect(headers).toHaveProperty('access-control-allow-headers');
+      expect(headers).toHaveProperty('access-control-allow-methods');
```

Comment:
I think it is excellent to check if all properties are available, but you could also use:
```
expect(headers).toMatchObject({
  'access-control-allow-origin': expect.any(String),
  'access-control-allow-header': expect.any(String)
})
```

This is just a suggestion, does not need to change.